packages/ra-ui-materialui/src/input/ImageInput.tsx

```diff
@@ -9,7 +9,6 @@ export const ImageInput = (props: ImageInputProps) => {
         <StyledFileInput
             labelMultiple="ra.input.image.upload_several"
             labelSingle="ra.input.image.upload_single"
-            classes={ImageInputClasses}
```

Comment:
why?

Reply:
It's something I should have removed when migrating to mui5